src/util/file_util.h

```diff
@@ -15,8 +15,10 @@ Author: Daniel Kroening, kroening@kroening.com
 // C++17 will allow us to use std::filesystem::path::remove_all
 void delete_directory(const std::string &path);
 
-// C++17 will allow us to use std::filesystem::current_path
+// C++17 will allow us to use std::filesystem::current_path (for both get and
+// set)
 std::string get_current_working_directory();
+void set_working_directory(const std::string &path);
 
```

Comment:
I would probably call that function set_current_path, to mimick the naming in C++17.

Reply:
Done!